### DISCUSSION TITLE
Prisma really needs something like whereRaw or selectRaw.

### DISCUSSION THREAD

**Discussion by SkullCutter02** [prisma/prisma#12548] [Q&A]
I have a use case where I need to do some fuzzy searching, which Prisma currently doesn't support. Without something like whereRaw, I need to resort to using a raw query, which makes me lose all sorts of type safety and convenience, when the where clause is literally the only part that requires me to use a raw query.

A lot of other ORMs like TypeORM and MikroORM have this functionality I believe, so it's quite jarring that Prisma doesn't have features like this.

**Comment by austincrim** [comment]
Hey @SkullCutter02 👋 ,

We have an [outstanding issue](https://github.com/prisma/prisma/issues/5560) for this feature. It would be great if you could add your +1 and use case to help us prioritize and potentially design the feature down the road.

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
